Gemfile

```diff
@@ -10,12 +10,12 @@ gem 'jwt'
 # Metadata presenter - if you need to be on development you can uncomment
 # one of these lines:
 # gem 'metadata_presenter',
-#    github: 'ministryofjustice/fb-metadata-presenter',
-#    branch: 'branching-OR-AND'
+#  github: 'ministryofjustice/fb-metadata-presenter',
+#  branch: 'branching-checkboxes-OR-AND'
 # gem 'metadata_presenter', path: '../fb-metadata-presenter'
 #
 gem 'fb-jwt-auth', '0.7.0'
-gem 'metadata_presenter', '1.7.1'
+gem 'metadata_presenter', '1.7.2'
```

Comment:
should this use 1.7.3?

Reply:
I merged the 1.7.3 some minutes ago. If that's okay I prefer to use the 1.7.2 here because both PRs are related and bump to 1.7.3 on my next PR (the check your answers).